lib/cliapp/src/Obelisk/CliApp/Process.hs

```diff
@@ -181,5 +183,10 @@ reconstructCommand :: Process.CmdSpec -> Text
 reconstructCommand (Process.ShellCommand str) = T.pack str
 reconstructCommand (Process.RawCommand c as) = processToShellString c as
   where
-    processToShellString cmd args = T.unwords $ map quoteAndEscape (cmd : args)
-    quoteAndEscape x = "'" <> T.replace "'" "'\''" (T.pack x) <> "'"
+    processToShellString cmd args = T.unwords $ map (shellQuoteAndEscapeSingle . T.pack) (cmd : args)
+
+shellQuoteAndEscapeSingle :: Text -> Text
```

Comment:
Is there a more robust shell-escaping strategy we could use instead? I found http://hackage.haskell.org/package/shell-escape and http://hackage.haskell.org/package/posix-escape, but neither of those packages seem to have been updated recently.

Reply:
Great question. Mind if I open a ticket for this and bump it to a different PR?

Reply:
However, if we went with the "add things to .ghci" design then it would make *this particular feature* skirt the problem.

Reply:
Sure, I'm not expecting you to rework existing code.

Reply:
#476